google-cloud-spanner/clirr-ignored-differences.xml

```diff
@@ -406,4 +406,10 @@
     <className>com/google/cloud/spanner/AbstractLazyInitializer</className>
     <method>java.lang.Object initialize()</method>
   </difference>
+  
+  <difference>
+    <differenceType>7013</differenceType>
+    <className>com/google/cloud/spanner/InstanceInfo$Builder</className>
```

Comment:
This change might break customers and probably should have a major version bump.

Reply:
The added method was in an unnecessary abstract builder class. I've removed the abstract class and renamed the concrete subclass to be equal to this abstract class. This way the change is non-breaking. The entry has been removed from ignored-diffs.

Reply:
Added a default implementation to the method in order to avoid the breaking change.